src/api/item.ts

```diff
@@ -1,22 +1,55 @@
+'use server';
+
+import { revalidateTag } from 'next/cache';
+import { cookies } from 'next/headers';
+
 import type { ItemType } from '@app/types/list.types';
 
+export async function getSubcategoryItems(
+  id: string
+): Promise<{ subcategoryItems: ItemType[] } | undefined> {
+  try {
+    const token = cookies().get('token');
+    if (token && token.value) {
```

Comment:
Good! In my pull request categories, I have introduced a function that returns token value, so when you merge we need to reuse it everywhere:
 const token = await getUserToken () instead of  const token = cookies().get('token');
    if (token && token.value) .